src/Standards/Squiz/Tests/WhiteSpace/OperatorSpacingUnitTest.1.inc.fixed

```diff
@@ -182,7 +182,7 @@ function foo(string $bar, array $baz, ?MyClass $object) : MyClass {}
 
 declare(strict_types=1);
 
-function foo($c = ((BAR) ? 10 : 100)) {}
+function foo($c = ((BAR)?10:100)) {}
```

Comment:
I'm undecided about this change. The `?` and `:` symbols _are_ operators in this context, but the sniff is specifically skipping over the whole function parameter list. From what I can tell from the comments, the sniff has always intended to skip these, but the implementation didn't work that way until now.

Reply:
Good point. Hm... in that case, maybe we should just skip over declare statements and keep the pre-existing logic for function declarations only ?

I also wonder if we could only apply that logic for the `T_EQUAL` for default values ? I have a feeling the `T_MINUS` in the condition is an outdated condition, which has been superseded by improved handling of unary tokens in the later part of the `isOperator()` method.